pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy

```diff
@@ -57,11 +57,13 @@ class Root implements Serializable {
 
     Libraries libraries
 
+    Axes axes
+
     final String astUUID
 
     @Whitelisted
     Root(Agent agent, Stages stages, PostBuild post, Environment environment, Tools tools, Options options,
-         Triggers triggers, Parameters parameters, Libraries libraries, String astUUID) {
+         Triggers triggers, Parameters parameters, Libraries libraries, Axes axes, String astUUID) {
```

Comment:
As I've mentioned, I think this should just be a `stage` level thing.

Reply:
I'll wait for the other PR and move it to nested stages